### ISSUE TITLE
Quality of life issues

### ISSUE THREAD

**Issue by DavidUdell** [DavidUdell/sparse_circuit_discovery#121]
1. Print newlines between the separate `tqdm` loading blocks.
2. Put `NUM_UP_NODES` _above_ `NUM_DOWN_NODES` in `central_config.yaml`.
3. Strip out `wandb` logging where it doesn't play a meaningful role.

**Comment by DavidUdell** [comment]
Note that it's nice to have `wandb` up front in `collect_acts.py`, so nothing waits on your wandb credentials login in a  pipeline run.

**Comment by DavidUdell** [comment]
Improved documentation and the `tqdm` stdout; the current `wandb` usage is fine, on reflection.
